FSJ_django20_project/FSJ/forms_award.py

```diff
@@ -0,0 +1,31 @@
+from .models_award import Award
+from django.forms import ModelForm, CheckboxSelectMultiple
+
+class AwardForm(ModelForm):
+
+    class Meta:
+        model = Award
+        exclude = ()
+        #uncomment this out when awards are ready for multiple years of study
```

Comment:
Awards don't accept multiple years of study because they're an integer field. Is the implementation of multiple years waiting on something specific?

Reply:
I've experimented with a ManyToMany relationship with a new YearOfStudy model but it wasn't stable, the version here is the latest, stable version

Reply:
Ah, I see, okay I think it's okay to leave as a place holder the way we've done with Program then.